.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: go
+
+go:
+- 1.11.x
+- master
+
+go_import_path: github.com/NathanielRose/bedrock
+
+before_install:
+- echo $clientID
+- echo $clientSecret
+- echo $publicKey
+- echo publicKey
+- wget https://releases.hashicorp.com/terraform/0.11.11/terraform_0.11.11_linux_amd64.zip

```

Comment:
Let's put this into a ENV_VAR so it's easy to update the version without having to modify the travis file